.scala-steward.conf

```diff
@@ -8,6 +8,7 @@ updates.ignore = [
 updates.pin = [
   # To be updated in tandem with upstream Akka
   {groupId = "com.fasterxml.jackson.core", version = "2.10."}
+  {groupId = "com.fasterxml.jackson.datatype", version = "2.10."}
```

Comment:
Isn't `master` for Akka 2.6.x only? Then it should align the Jackson version (in Dependencies.scala) with what we have in Akka, which is 2.11.4

Also, see https://github.com/akka/akka/blob/master/.scala-steward.conf for pin and ignore